cleanup/README.md

```diff
@@ -0,0 +1,14 @@
+# Setup environment
+
+Action cleans up workspace directory after previous run. The main reason to add
+this action is [tarantool/tarantool-qa#145](https://github.com/tarantool/tarantool-qa/issues/145). 
+When submodules have changed, the standard checkout fails with git errors. 
+The actions/checkout error still opened
```

Comment:
1. Please wrap `actions/checkout` into backtick symbols.
2. `error still opened` -> `error is still opened`

Reply:
done